src/main/resources/org/jenkinsci/plugins/configfiles/ConfigFilesManagement/selectprovider.properties

```diff
@@ -18,4 +18,4 @@
 
 selectprovider=Select the file type you want to create
 filetype=Type
-
+configIdSuffixDescription=Config file ID is prefixed by the provider name (e.g. "org.jenkinsci.plugins.configfiles.maven.MavenSettingsConfig") and suffixed by the given "ID suffix" or by a random number if no "ID Suffix" is specified

```

Comment:
@jglick yes but the delete operation only knows the ID, we would have to add an additional lookup :-(